nRF9160Master/src/BMS/BMHandler.c

```diff
@@ -97,6 +96,13 @@ float ReadI2CVoltage(void)
     float fVolt = (usVData * 2.44) / 1000.0;
 
     return fVolt;
+    }
+    else 
+    {
+        printk("DEBUG : PMIC is not ready for reading Voltage\n\r");
+        // return 0;
```

Comment:
Why the error case is not handled here ? else case should return 0 here so that we can check the result is valid or not, if not valid value will be zero.